City-of-Helsinki/drupal-helfi-kymp
# UHF-5523: KYMP list of plans

### What has been done
- created a new paragraph "list of plans" that lists items from an external RSS feed
- introduced HDBT subtheme to KYMP

### How to install
- set up KYMP instance
- checkout this branch: `git checkout UHF-5523_list_of_plans`
- clear cache: `make drush-cr`
- import config: `make drush-cim`
- update translations (in the container, `make shell`): `drush locale:check && drush locale:update && drush cr`

### How to test
- create a basic page or a landing page and add "list of plans" paragraph on it
- give the paragraph a title and a description
- save the page and see the items from the RSS feed (feed URL: https://ptp.hel.fi/rss/nahtavana_nyt/)
- create language versions of the page and check if the translations work
